pandas/tseries/resample.py

```diff
@@ -686,6 +686,10 @@ def _downsample(self, how, **kwargs):
         if not len(ax):
             # reset to the new freq
             obj = obj.copy()
+            if how == "size" and isinstance(obj, pd.DataFrame):
```

Comment:
this is not a good way of fixing. you will have to trace thru the ``.size`` implementation.